packages/shared/dashboards/status-card/health-item.spec.tsx

```diff
@@ -0,0 +1,141 @@
+import * as React from 'react';
+import { HealthState } from '@openshift-console/dynamic-plugin-sdk';
+import { render, screen } from '@testing-library/react';
+import HealthItem from './HealthItem';
+
+jest.mock('@odf/shared/status/SecondaryStatus', () => () => (
+  <div data-test-id="secondary-status" />
+));
+
+const popOverComponent = jest.fn();
+jest.mock('@patternfly/react-core', () => ({
+  Popover: (props) => {
+    popOverComponent(props);
+    return <div data-test-id="mocked-pop-over" />;
+  },
+  Button: () => <div data-test-id="mocked-button" />,
+  PopoverPosition: {
+    top: 'mocked-pop-over-position-top',
+  },
+}));
+
+jest.mock('./states', () => ({
+  healthStateMapping: {
+    [HealthState.OK]: {
+      icon: 'unit-test-ok-icon',
+    },
+    [HealthState.UNKNOWN]: {
+      icon: 'unit-test-unknown-icon',
+    },
+  },
+  healthStateMessage: () => 'mocked-health-state-message',
+}));
+
+const mockedFunction = jest.fn().mockImplementation(() => undefined);
+jest.mock('@odf/shared/useCustomTranslationHook', () => ({
+  useCustomTranslation: () => {
+    return { t: mockedFunction };
+  },
+}));
+
+const title = 'unit-tests';
+
+describe('tests for document elements of health item component', () => {
+  it('should render and have a span saying loading title status when in loading', () => {
+    render(<HealthItem title={title} state={HealthState.LOADING} />);
+    expect(mockedFunction).toHaveBeenCalledWith('Loading {{title}} status', {
+      title,
+    });
```

Comment:
IMHO this is testing the translation hook so we can place this test in the translation hook test suite. Instead, waht about testing that the component has been rendered?
```suggestion
    expect(
      container.querySelector(`[data-item-id="${title}-health-item"]`)
    ).toBeTruthy();
```